.gitignore

```diff
@@ -0,0 +1,19 @@
+**/*.pyc
+**/*.so
+**/*.egg-info
+**/*.whl
+/dist/
+.ipynb_checkpoints
+.DS_Store
+.mypy_cache/
+.pytype/
+.idea
+.vscode
+.envrc
+
+# virtualenv/venv directories
+/venv/
```

Comment:
Are these absolute paths? Like when would folders like bin, include, etc. show up in this git repo?